src/components/NavigationLink/NavigationLink.tsx

```diff
@@ -21,6 +21,12 @@ export interface NavigationLinkProps extends NestableBaseComponentProps {
    */
   newWindow?: boolean;
 
+  /**
+   * If present, indicates the link refers to the current page. This is usually used in cases where the link
+   * is visually styled to indicate it is selected or is the current page, such as in a navigation list.
+   */
+  ariaCurrent?: 'page';
```

Comment:
do we have any reason to allow its other values? page, step, location, date, time

Reply:
Not at this time; page is the only one with a current concrete use-case.